pkg/proto/configuration/global/global.proto

```diff
@@ -15,9 +15,16 @@ message JaegerConfiguration {
 
   // OpenTracing service name.
   string service_name = 3;
+}
 
-  // Whether or not all traces should be sampled.
-  bool always_sample = 4;
+message StackdriverConfiguration {
+  // GCP project to send trace data to
```

Comment:
Maybe it's a bit more elegant to not end this sentence with a preposition? Also make sure that all sentences are ended with a period.

> GCP project to which trace data is sent. Can be left blank when running on GCP, as it is automatically determined by the metadata server.

Reply:
Done